docs/dev/release.rst

```diff
@@ -6,8 +6,13 @@ This document outlines the steps for releasing a versioned JDAViz package to
 PyPI. Currently, these do not cover submitting package updates to the
 ``astroconda`` channel.
 
-This process currently requires high-level access to the JDAViz repository,
-as it relies on the ability to commit to master directly.
+This process currently requires admin-level access to the JDAViz repository,
+as it relies on the ability to commit to master directly. To do this, it is
```

Comment:
```suggestion
as it relies on the ability to commit to master directly. If you are not confident 
you know how to avoid accidentally pushing something unexpected it is
```